scripts/logs/logs-pretty.sh

```
@@ -0,0 +1,159 @@
+        line=$(echo "$line" | sed "s/\[INFO\]/${GREEN}[INFO]${NC}/g")
+    elif echo "$line" | grep -q "\[DEBUG\]"; then
+        line=$(echo "$line" | sed "s/\[DEBUG\]/${GRAY}[DEBUG]${NC}/g")
+    fi
+    
+    # Format timestamps
+    line=$(echo "$line" | sed -E "s/([0-9]{4}\/[0-9]{2}\/[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2})/${BLUE}\1${NC}/g")
+    
+    # Try to pretty-print JSON if jq is available
+    if $USE_JQ && echo "$line" | grep -q '{.*}'; then
+        # Extract the JSON part
+        local json_part=$(echo "$line" | sed -E 's/.*(\{.*\}).*/\1/')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)